client/consensus/babe/src/lib.rs

```diff
@@ -995,11 +1046,15 @@ impl<Block, Client, Inner> BlockImport<Block> for BabeBlockImport<Block, Client,
 
 			let viable_epoch = epoch_changes.viable_epoch(
 				&epoch_descriptor,
-				|slot| self.config.genesis_epoch(slot),
+				|slot| Epoch::genesis(&self.config, slot)
 			).ok_or_else(|| {
 				ConsensusError::ClientImport(Error::<Block>::FetchEpoch(parent_hash).into())
 			})?;
 
+      let epoch_config = next_config_digest.unwrap_or_else(
```

Comment:
```suggestion
			let epoch_config = next_config_digest.unwrap_or_else(
```

Reply:
Ah! Not sure why my editorconfig suddenly failed here.